tests/test_pktgen.py

```diff
@@ -119,13 +126,21 @@ def test_pktgen(duthosts, enum_dut_hostname, enum_frontend_asic_index, tbinfo, l
     15000 packets were expected but only {} found".format(port, 15000-int(interf_counters)))
 
     # Check CPU util after sending traffic
-    cpu_after = duthost.shell("show proc cpu --verbose | sed '1,/CPU/d' | awk '{print $9}'")["stdout_lines"]
+    cpu_after = duthost.shell(CPU_CMD)["stdout_lines"]
     for entry in cpu_after:
         pytest_assert(
             float(entry) < cpu_threshold,
-            "Cpu util was above threshold {} for atleast 1 process"
+            "Cpu util was above threshold {} for pktgen process"
```

Comment:
I can see the cpu util is more likely go up to 100% for pktgen which it is running, so check the cpu utilization is more easy to see the error.
may need to check the syslog/dmesg to see any signature says there are cpu lockup issue.

Reply:
Syslogs are being checked via loganalyzer and dmesg is being checked in line 136-142. Can we remove the CPU util check?

Reply:
yes, I think so, just remove the CPU utlization checks